{
  "_annotation": {
    "cve_id": "CVE-2022-21607",
    "generated_from": "https://raw.githubusercontent.com/anchore/cve-data-enrichment/main/data/anchore/2022/CVE-2022-21607.json",
    "reason": "Improve version ranges to indicate fix"
  },
  "cve": {
    "configurations": [
      {
        "nodes": [
          {
            "cpeMatch": [
              {
                "criteria": "cpe:2.3:a:oracle:mysql:*:*:*:*:*:*:*:*",
                "matchCriteriaId": "533F49AE-F211-5A66-9673-06858B1FDAA5",
                "versionEndExcluding": "8.0.29",
                "vulnerable": true
              }
            ],
            "negate": false,
            "operator": "OR"
          },
          {
            "cpeMatch": [
              {
                "criteria": "cpe:2.3:a:oracle:mysql_server:*:*:*:*:*:*:*:*",
                "matchCriteriaId": "101323D0-E3D9-534C-9520-EA821F48BB0B",
                "versionEndExcluding": "8.0.29",
                "vulnerable": true
              }
            ],
            "negate": false,
            "operator": "OR"
          }
        ]
      }
    ]
  }
}